### DISCUSSION TITLE
Cleanup Temporary Options File does not work

### DISCUSSION THREAD

**Discussion by malconvsilva** [spatie/browsershot#811] [Bugs]
The function cleanupTemporaryOptionsFile does not work when successfully generating the pdf when using the writeOptionsToFile function.
When checking the Brosershot.php file there is a "return" on line 1029 and the cleanupTemporaryOptionsFile occurs on line 1032. Therefore, the function to clean TemporaryOptionsFile never happens when the process is successful.

I'm using version 3.61.0, but I saw that in version 4.0.1 clenup also occurs after "return" on line 1049.

**Comment by freekmurze** [comment]
Feel free to send a PR to fix this 👍
